server/src/routes.py

```diff
@@ -46,15 +51,19 @@ def setup_routes(app):
                 json_msg = aiohttp.WSMessage.json(msg)
                 if json_msg['fname'] == 'closeSocket':
                     print('Closing socket')
+                    message = MessageSingleton("sniffer socket closed")
```

Comment:
this doesn't work for me either, although neither does the print line above. seems to go straight to [line 65](https://github.com/kaganjd/sealion/pull/49/files#diff-a0d3c251a1938d8bbe54da486bc1beadL65), which still communicates that the socket is closing. is this what happens for you? if so, i think we should:
- remove `print('Closing socket')` and the MessageSingleton here
- try adding the MessageSingleton to right after [line 65](https://github.com/kaganjd/sealion/pull/49/files#diff-a0d3c251a1938d8bbe54da486bc1beadL65)